### ISSUE TITLE
Book 05: Sentence 084

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#19]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/1c00ae97-f7aa-4078-96a0-d9ef1054f090)

```
# sent_id = Book_05_-_The_Omens.txt:84
# text = Ic matlactli omome capitulo; vncan mjtoa: in occentlamantli tetzaujtl catca: in jtoca catca: tlacanexqujmjlli.
# text[norm] = Ic matlactli omome capítulo; oncan mihtoa: in occentlamantli tetzahuitl catca: in itoca catca: tlacanexquimilli.
# text[orig] = Ic·matlactli·omome·capitulo·;·vncan·mjtoa·:·in·occentlamantli¶tetzaujtl·catca·:·injtoca·catca·:·tla¶canex·qujmjlli·.
# text[eng] = Twelfth chapter: where it is said: that another omen there was: that its name was: bundle of ashes.
# text[osp] = Capitulo doze, que trata de vnas fantasmas: que aparecen de noche, que llaman tlacanexqujmilli.
# references = A&D/177:1
# tagged = 100.00%
# lemmatised = 100.00%
# analysed = 100.00%
# heads = 94.74%
# relations = 31.58%
1	Ic	ic	ADV	_	_	2	_	_	Folio=11v|Paragraph=4|Line=1|Norm=Ic
2	matlactli	matlactli	NUM	_		4	_	_	Folio=11v|Paragraph=4|Line=1|Norm=matlactli|Gloss=diez|Analysed=Yes
3	omome	ome	NUM	_	_	2	_	_	Folio=11v|Paragraph=4|Line=1|Norm=omome|Gloss=y.dos
4	capitulo	capítulo	NOUN	_	State=Abs	7	_	_	Folio=11v|Paragraph=4|Line=1|Norm=capítulo|Analysed=Yes
5	;	;	PUNCT	_		7	punct	_	Folio=11v|Paragraph=4|Line=1|Norm=;|Analysed=Yes
6	vncan	oncan	ADV	_		7	_	_	Folio=11v|Paragraph=4|Line=2|Norm=oncan|Gloss=aquí|Analysed=Yes
7	mjtoa	ihtoa	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Reflexive[obj]=Yes|Subcat=Tran|Tense=Pres|VerbForm=Fin	0	root	_	Folio=11v|Paragraph=4|Line=2|Norm=mihtoa|Analysed=Yes
8	:	:	PUNCT	_		11	punct	_	Folio=11v|Paragraph=4|Line=2|Norm=:|Analysed=Yes
9	in	in	DET|SCONJ	_	_	11	_	_	Folio=11v|Paragraph=4|Line=2|Norm=in
10	occentlamantli	occentlamantli	NOUN	_	State=Abs	11	_	_	Folio=11v|Paragraph=4|Line=2|Norm=occentlamantli|Analysed=Yes
11	tetzaujtl	tetzahuitl	NOUN	_	State=Abs	_	_	_	Folio=11v|Paragraph=4|Line=3|Norm=tetzahuitl|Analysed=Yes
12	catca	catqui	VERB	_	Subcat=Intr	11	_	_	Folio=11v|Paragraph=4|Line=3|Norm=catca
13	:	:	PUNCT	_		15	punct	_	Folio=11v|Paragraph=4|Line=3|Norm=:|Analysed=Yes
14	in	in	DET|SCONJ	_	_	15	_	_	Orig=injtoca|Folio=11v|Paragraph=4|Line=3|Norm=in
15	jtoca	toca	NOUN	_	_	11	_	_	Orig=injtoca|Folio=11v|Paragraph=4|Line=3|Norm=itoca|Gloss=su.nombre
16	catca	catqui	VERB	_	Subcat=Intr	15	_	_	Folio=11v|Paragraph=4|Line=3|Norm=catca
17	:	:	PUNCT	_		18	punct	_	Folio=11v|Paragraph=4|Line=3|Norm=:|Analysed=Yes
18	tlacanexqujmjlli	tlacanexquimilli	NOUN	_	State=Abs	15	_	_	Orig=tla¶·canex·qujmjlli|Folio=11v,11v,11v|Paragraph=4,4,4|Line=3,4,4|Norm=tlacanexquimilli|Analysed=Yes
19	.	.	PUNCT	_		7	punct	_	Folio=11v|Paragraph=4|Line=4|Norm=.|Analysed=Yes
```

**Comment by ftyers** [comment]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/71a76664-c34b-4812-9620-20bd0a722969)

Updated according to decisions:
- Ordinals: 001 (#14)

Remaining questions:
- What to do with compound numbers, English treebanks use `compound` (but with hyphens *fifty-two*), in Spanish they are written together and in Russian they are done with `nummod`. The guidelines used to mention `compound` but it appears that it was removed for v2. 
- What is the POS of *occentlamantli* ? Is it a `NOUN`, should it be split, into *oc centlamantli*? (Probably yes given the frequency list information below). 
  - If so, *centlamantli* should be a `NUM` + `NumType=Clf` ? 
- What to do with *catca* 
- Which words are subjects and which are predicates? -_-


Frequency list for `oc ?cent`:
```
     26 oc centlamantli
     12 oc centetl
      3 oc centlamãtli
      2 oc centlamãtin
      2 oc centlamantin
      1 occentlamatli
      1 occentlamantli
      1 oc centlapal
      1 oc centlamatli
      1 oc centlamanti
```

**Comment by ftyers** [comment]
> > What is the POS of occentlamantli ? Is it a NOUN, should it be split, into oc centlamantli? (Probably yes given the frequency list information below).
> 
> what about _oc ce in tlamantli_ ?

I can't find any examples of that in the corpus. Do you suggest that stuff like *centetl* should also be *ce in tetl*? I'm not completely down on that, but what would the structure be?

**Comment by Lguyogiro** [comment]
Yeah I guess the downside would be the head of the np would be the classifier I guess, e.g. `ce  -advmod-> oc`, `tlamantli -nummod-> ce`, `tlamantli -det-> in`, `tlamantli -nmod-> tetzahuitl`...or something like that.

**Comment by ftyers** [comment]
How about this:
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/d946694c-3ade-479c-b353-1cf448e92fbb)

**Comment by Lguyogiro** [comment]
LGTM

**Comment by ftyers** [comment]
Resolved in 4419742.
